### PR TITLE
feat(workflows): Add PR auto-labeler

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [x] Tested your changes against relevant architectures and platforms;
  - [x] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,204 @@
+# Area labels based on the path of the file
+'area/.github':
```
Comment (antoineco) [inline .github/labeler.yml:2]: I don't think any of these quotes are required.

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+- .github/*
+- .github/**/*
+
+'area/.github/workflows':
+- .github/workflows/*
+
+
```
Comment (antoineco) [inline .github/labeler.yml:9]: One new line should be sufficient.

File: .github/workflows/label-prs.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v4
+      with:
+          sync-labels: true
```
Comment (antoineco) [inline .github/workflows/label-prs.yaml:17]: Something went wrong with the indentation here.

**Comment by craciunoiuc** [comment]
I remember @nderjung saying he wants a different approach for this so will just close

**Pull request closed without merging**